sampleconfig/orderer.yaml

```diff
@@ -68,14 +68,14 @@ General:
         # ServerPrivateKey defines the file location of the private key of the TLS certificate.
         ServerPrivateKey:
 
-    # Genesis method: The method by which the genesis block for the orderer
+    # Bootstrap method: The method by which to obtain the bootstrap block
     # system channel is specified. The option can be one of:
     #   "file" - path to a faile containing the genesis block or config block of system channel
     #   "none" - allows an orderer to start without a system channel configuration
-    GenesisMethod: file
+    BootstrapMethod: file
 
     # Bootstrap file: The file containing the bootstrap block to use when
-    # initializing the orderer system channel and GenesisMethod is set to
+    # initializing the orderer system channel and BootstrapMethod is set to
     # "file".  The bootstrap file can be the genesis block, and it can also be 
     # a config block for late bootstrap of some consensus methods like Raft.
     BootstrapFile:
```

Comment:
for the same reason, the sample value for this field will need to be re-added after PR to fabric-samples being merged https://github.com/hyperledger/fabric-samples/pull/77

Reply:
fabric-sample is updated.